cobc/scanner.l

```diff
@@ -1310,12 +1316,51 @@ error_literal (const char *type, const char *literal, unsigned int literal_error
 	cb_error ("%s", err_msg);
 }
 
+/* Scans a symbolic EBCDIC character given as a sequence of decimal digits
+   starting with `c`, and returns the corresponding code as per the machine
+   character set (ASCII or EBCDIC).  */
+static cob_u8_t
+scan_ebcdic_char (int c)
+{
+	char buff[10];	       /* Arbitrary limit, mostly for error-reporting */
+static  cob_u8_t ebcdic_to_ascii[256] ;
+static  int ebcdic_to_ascii_initialized = 0 ;
+
+	unsigned int j = 0;
+	do {
+		buff[j++] = c;
+	} while ((c = input ()) != EOF && '0' <= c && c <= '9' && j < 10);
+	unput (c);
+	buff[j++] = 0;
+	c = atoi (buff);
+	if (c < 1 || c >= 257) {
+		cb_error (_("symbolic character %s out of allowed EBCDIC range (1...256)"),
```

Comment:
please reword that msgid to make it usable also for unicode and others later (drop EBCDIC, keep range, possibly adjust "symbolic character")